gradle.properties

```diff
@@ -18,9 +18,9 @@
 buildKotlinVersion=1.3.50-dev-44
 buildKotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_Compiler),number:1.3.50-dev-44,pinned:true/artifacts/content/maven
 remoteRoot=konan_tests
-kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_Compiler),number:1.3.50-dev-259,branch:default:true,pinned:true/artifacts/content/maven
-kotlinVersion=1.3.50-dev-259
-testKotlinVersion=1.3.50-dev-259
+kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_Compiler),number:1.3.50-dev-325,branch:default:false,pinned:false/artifacts/content/maven
+kotlinVersion=1.3.50-dev-325
+testKotlinVersion=1.3.50-dev-325
```

Comment:
These are preview version changes, will update to master build, when the corresponding branch is merged in Kotlin